lib/workers/repository/dependency-dashboard.ts

```diff
@@ -97,7 +98,7 @@ export async function ensureMasterIssue(
     issueBody += '\n';
   }
   const errorList = branches.filter(
-    (branch) => branch.res && branch.res.endsWith('error')
+    (branch) => branch.res && branch.res === ProcessBranchResult.Error
```

Comment:
I tried to make this PR as "mechanical" as possible, this line will be fixed after eslint restriction